ansible/roles/k3s/tasks/main.yml

```diff
@@ -0,0 +1,77 @@
+---
+
+- name: Check for existing k3s installation
+  stat:
+    path: /var/lib/rancher/k3s
+  register: stat_result
+
+- name: Download and air-gapped installation of k3s
```

Comment:
```suggestion
- name: Download and air-gapped installation of k3s
  # Using air-gapped install so containers are pre-installed to avoid rate-limiting from registries on cluster startup
```

Is that right?